### PR TITLE
Checks for login field

### PR DESCRIPTION
**Pull request by Pritom14**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Pritom14** [comment]
@therajanmaurya Sir, can you review this

File: app/src/main/java/org/mifos/selfserviceapp/presenters/LoginPresenter.java [hunk 1]
```diff
@@ -75,8 +75,9 @@ public void detachView() {
      */
     public void login(final String username, final String password) {
         checkViewAttached();
-        getMvpView().showProgress();
+
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/selfserviceapp/presenters/LoginPresenter.java:78]: Remove this blank line.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by Pritom14** [comment]
Done Sir @therajanmaurya

**Review by therajanmaurya — APPROVED** [review]

**Comment by therajanmaurya** [comment]
@Pritom14 Please squash your two commits and follow the Commit Style guide https://github.com/openMF/self-service-app/wiki/Commit-Style-Guide

**Pull request merged**